tests/test_ndmg_dwi_pipeline.py

```diff
@@ -6,10 +6,91 @@
 import ndmg
 from ndmg.utils.cloud_utils import s3_get_data
 from pathlib import Path
+from ndmg.utils.gen_utils import create_datadescript, DirectorySweeper
+import tempfile
 
 KEYWORDS = ["sub", "ses"]
 
 
+@pytest.fixture
+def input_dir_tree(tmp_path):
+    data = {"002547": 2, "002548": 1, "002449": 3}
+    for sub, session in data.items():
+        for ses in range(1, session + 1):
```

Comment:
so this will make the directories for...
```
0025427 1
0025427 2
0025427 1
002449 1
002449 2
002449 3
```?

Why don't we just do something like
`data = {"0025427": [1, 2], "002548": [1], ...}`
instead, and then iterate through each list value? makes it a bit more clear I think